google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java

```diff
@@ -146,6 +147,15 @@ public static ListOption pageSize(int pageSize) {
     return new PageSizeOption(pageSize);
   }
 
+  /**
+   * If this is for a partitioned read & query and this field is set to `true`, the request will be
+   * executed via Spanner independent compute resources.
+   */
+  @BetaApi
```

Comment:
Apart from this annotation, let's add a comment above to let users know that this option is available in Beta mode (and is not generally available).